modules/react/checkbox/README.md

```diff
@@ -1,133 +1,7 @@
 # Canvas Kit Checkbox
 
-A checkbox input.
+View the
+[documentation for Checkbox](https://workday.github.io/canvas-kit/?path=/docs/components-inputs-checkbox-react--default-story)
+on Storybook.
 
 [> Workday Design Reference](https://design.workday.com/components/inputs/checkboxes)
-
-## Installation
-
-```sh
-yarn add @workday/canvas-kit-react
-```
-
-## Usage
-
-#### Simple Example
-
-**Note:** While a base checkbox component is provided in this package, it is **not accessible** when
-used as is. It should be used in tandem with [`FormField`](../../form-field/react) to be made fully
-accessible (see below).
-
-```tsx
-import * as React from 'react';
-import Checkbox from '@workday/canvas-kit-react/checkbox';
-
-<Checkbox disabled={false} checked={checked} onChange={this.handleCheck} />;
-```
-
-#### Accessible Example
-
-```tsx
-import * as React from 'react';
-import Checkbox from '@workday/canvas-kit-react/checkbox';
-import FormField from '@workday/canvas-kit-react/form-field';
-
-<FormField label="My Field" inputId="my-checkbox-field">
-  <Checkbox disabled={false} checked={checked} onChange={this.handleCheck} id="my-checkbox-field" />;
-</FormField>;
```

Comment:
Any reason why we dropped this in the MDX docs? All of the other inputs note `FormField` usage for a11y.

Reply:
I left it out as I was rewriting the Checkbox docs on `canvas-site` since it seemed redundant to have a FormField on a Checkbox given that it already has a label. Meant to look into this further, but haven't had time -- I've gone ahead and added FormField back to the Checkbox examples.

I didn't add it to the [Indeterminate example](https://5d854c26ba934e0020f5e98a-duglscczpo.chromatic.com/?path=/docs/components-inputs-checkbox-react--indeterminate), though, because I couldn't think of a good label for my example with the pizza toppings. Any ideas?

Reply:
Good point. The checkbox label has a `for` attribute, so it should be fine a11y-wise. Maybe let's leave it out for all the examples, but either mention that it can be used with FormField or provide an example for it?

Reply:
Done. I've included a mention that Checkbox can be used with or without Form Field, and there are a few examples which use Form Field: Label Position, Required, and the Error States.